shared/openId4VCI/Utils.ts

```diff
@@ -92,3 +92,20 @@ export const getJWT = async context => {
     throw e;
   }
 };
+export const VC_DOWNLOAD_TIMEOUT = 30;
```

Comment:
can we move this property to inji-default.properties

Reply:
yes, let's move it to inji-default.properties and add as Initial config which is being used for caching

Reply:
Change included